codec_specs.md

```diff
@@ -463,8 +463,10 @@ Description: VP9 is an open and royalty free video compression format developed
 
 Codec BlockAdditions: A single-channel encoding of an alpha channel **MAY** be stored in `BlockAdditions`. The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
-Initialization: The `CodecPrivate` **MUST** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
-Due to the existence of some old non compliant muxers which were not providing `CodecPrivate`, the decoder **SHOULD** reconstruct it from the first frame in the case `CodecPrivate` is not present.
+Initialization: The `CodecPrivate` **SHOULD** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
+As many muxers don't provide the `CodecPrivate` for "V_VP9" so it's not a hard requirement. However, it is **RECOMMENDED** to provide this information that can help select a decoder on playback.
```

Comment:
IMO the second sentence is redundant (same meaning as the "SHOULD" one line above) and can be removed.

Reply:
They are not giving the same information. The first explain why it's not a MUST. The other why it should be a MUST but it's not.

Reply:
"However, it is **RECOMMENDED** to provide this information that can help select a decoder on playback." is synonym of "The `CodecPrivate` **SHOULD** contain".

Or I would rephrase for avoiding 2x the IETF wording (SHOULD & RECOMMENDED), overall it would be:
```suggestion
Initialization: The `CodecPrivate` **SHOULD** contain a list of specific VP9 codec features as described in the "VP9 Codec Feature Metadata" section of [@!WebMContainer].
This piece of data helps to select a decoder on playback, but as defacto many muxers don't provide the `CodecPrivate` for "V_VP9" so it's not a hard requirement.
It is possible for the decoder to reconstruct the "VP9 Codec Feature Metadata" from the first frame in the case `CodecPrivate` is not present.
```

Reply:
OK, I took your suggestion for the second sentence and slightly tweaked it.